### DISCUSSION TITLE
Huge problems for me

### DISCUSSION THREAD

**Discussion by fulltimer123** [forkineye/ESPixelStick#552] [Q&A]
I have somehow screwed my ESP up.

Have re-installed the V3 firmware via java install tool (USB) but now my router does not see it as a client so I cannot even get to the web interface.  Plus, plugging it in seems to put it into a loop where it's flash the first 8 or so pixels in the string 7 blue and 1 green every 15 seconds or so.

Have I cooked the chip??

**Comment by MartinMueller2003** [comment]
Most likely NO. You have not cooked the chip. Please send me your email address and I will send you a link to the latest installer with V4 (I know you are trying to use V3). V4 is compatible with the V3 hardware. The newer installer supports saving the serial output to a file. Flash the ESP and then save the log and attach the log file to this conversation.

**Reply by fulltimer123** [reply]
I have the v4 installation file, but the one I have doesn't even have the java install tool where it normally  should be; it's in the dist directory.  

I tried to run it anyhow, but it gave many "missing file"  errors. But here's a good email address for me  .......

gearjammer@bill-fischer.net      

Thanks.......     I'd LOVE to  get up on v4.

**Comment by MartinMueller2003** [comment]
The one that comes with the source code is way out of date. It will get updated with the next beta release. I sent you an invitation to the dist that I maintain as part of the alpha tester pool.

**Reply by fulltimer123** [reply]
Thank you  MM2003.  It took a bit of fiddling, but I'm now at the point where I was before my whole issue started,

Except I'm on V4!!!   

I don't see a way to load sequences directly to the board and since I'm on a single router for my whole house I can't connect (WIFI) at the moment.  Is Falcon software loaded on there??  Or how do I play sequences directly from the ESP (which I was told was possible with 4.x)??

**Comment by MartinMueller2003** [comment]
Sequences are stored on the SD card that you insert into the V3 vase card. 

- Put an SD card into an adapter on your computer
- copy the fseq files of interest to the SD card. 
- Eject the SD card
- plug the SD card into the V3 base card (with the power off)
- turn on the ESP. It should auto detect the SD card. 
- On the device config screen  go to input 2. 
- Select FPP Remote operation. 
- Scroll down to the FPP Remote config options
- in the drop down you will get a list of sequences that are detected on the SD card, Pick one. 
- Save the config and the sequence will play in a loop. 

FYI: You can also create a playlist (see the doc) and select a playlist instead of an individual sequence.

**Reply by fulltimer123** [reply]
Hi...   MM2003.....  Still having a problem with V4 where it disconnects my browser fairly often,  Any thoughts??

Also I've got the data line for my lights connected to D+; which I think is correct.    However the LED strips light up in random patterns/colors.

Finally, I can't see the stick as a client from my router (TPlink)  admin panel. 

Thanks again for all you've already done to help me.

**Comment by MartinMueller2003** [comment]
D+ is not enough. You also need V- from the ESP base card to the strip (no detour to the PSU).

Your router not recognizing the ESP is most likely why you see browser disconnects. Is the ESP rebooting as well? Keep in mind the ESP is configured to reboot if it cannot connect to a browser (You will want to turn that off for a show).

**Comment by fulltimer123** [comment]
I have  V+ V- and D+ configured to the the ESPI also find that the Java flashing to wants to flash on com (5) while device manager shows I only have Com(3).  I'm starting to think maybe my windows 10 installation is messed up;  I may rebuild the system from scratch.


UPDATE:   I did in fact rebuild Windows and everything is now working as expected, except for the random disconnects between the ESP and my browser.  Yahoo!!!!

**Reply by fulltimer123** [reply]
MM2003.......  still dropping WIFI connection.  Have emailed  support@forkineye.com,  but so far no answer.    Can you think of anything??  Have tried two different routers, no help......

**Comment by MartinMueller2003** [comment]
The ESPixelStick WiFi has been very stable for me on multiple platforms. We need to explore a few things. How far is it between the ESP and the router? What is (physically) between them? BTW, too close can also be an issue. When it is connected, what is the reported signal strength on the UI? What is coming out on the console when the connection drops?

**Reply by fulltimer123** [reply]
Thanks....  I think I may have stumbled on it last night.  I believe is a power issue as I noticed that with a pattern rainbow it disconnects and will not come back up.  Does it every time.

Temporarily, until I can get a bigger power  supply, I've selected a the chase pattern with ONE color  (255,0,0) and it has not dropped onetime  in 6 hours.   Hopefully this is finally the end of this thread, and I won't come begging for more help in the future.  

Thanks again (over and over).

**Reply by fulltimer123** [reply]
HI MM2003

As an experiment, I loaded current WLED on the chip. Cleared up the xLights show not running directly from my PC.

Also cleared up a problem with the external SD card socket being very intermittent.

Just a little feedback to maybe help YOU a little bit, for all the help you've given me.

**Comment by MartinMueller2003** [comment]
Thanks.
It still does not make sense since I am not seeing any of those issues.
Martin
    On Wednesday, August 31, 2022 at 02:50:02 PM EDT, fulltimer123 ***@***.***> wrote:  
 
 


HI MM2003

As an experiment, I loaded current WLED on the chip. Cleared up the xLights show not running directly from my PC.

Also cleared up a problem with the external SD card socket being very intermittent.

Just a little feedback to maybe help YOU a little bit, for all the help you've given me.

**Comment by MartinMueller2003** [comment]
I just updated the google drive dist.zip file. It has some bug fixes but not much else has changed.

**Reply by brmeadows2841** [reply]
> I just updated the google drive dist.zip file. It has some bug fixes but not much else has changed.
Is there a link to the google drive for espixelstick? If what is it?

**Comment by MartinMueller2003** [comment]
You create a playlist on any text editor, following the instructions in the readme. Then you upload the playlist to the SD card (through the UI or by putting the SD card in your pc). Lastly you set the FPP remote play to the name of the playlist. The ESP will now play the files in your playlist.
Martin
    On Wednesday, November 29, 2023 at 03:57:24 AM EST, paulrajan94 ***@***.***> wrote:  
 
 


Where is the option to create a playlist? @MartinMueller2003
